.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -8,26 +8,28 @@ assignees: ''
 ---
 # XRTK - Mixed Reality Toolkit Feature Request
 
+![](https://github.com/XRTK/XRTK-Core/raw/development/docs/logo.png)
+
 ## Is your feature request related to a problem? Please describe
 
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+<!-- A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] -->
 
 ## How would you classify your suggestion
 
-What type of enhancement is it, e.g:
+<!-- What type of enhancement is it, e.g:
 
 - New platform support
 - Usability / Configuration
-- Architecture / Services
+- Architecture / Services -->
```

Comment:
What do you think about keeping these things in and move the end of the comment tag to line 19?